mzbench/scenarios/fan_in.bdl

```diff
@@ -2,56 +2,68 @@
 
 #######
 # Scenario:
-# A single subscriber reading from "<MF_CHANNEL>/#" topic
-# 10000 publisher publishing to exclusive topic "<MF_CHANNEL>/<client_id>"
+# A single subscriber reading from "channels/<MF_CHANNEL_ID>/messages/#" topic
+# 10000 publisher publishing to exclusive topic "channels/<MF_CHANNEL_ID>/messages/<client_id>"
 # Overall msg rate: 10 000 msg/s
 # Message size: 100 random bytes
 # Running time: 5 min
 # QoS level: 2
-# NOTE: Update values in test, according to your values
-# <MF_MQTT_ENPOINT> = IP/domain of your MQTT endpoint
-# <MF_THING_1_ID> = First thing ID of pre-provisioned Mainflux thing
-# <MF_THING_1_key> = First thing key of pre-provisioned Mainflux thing
-# <MF_THING_2_ID> = Second thing ID of pre-provisioned Mainflux thing
-# <MF_THING_2_key> = Second thing key of pre-provisioned Mainflux thing.
-# <MF_CHANNEL> = pre-provisioned channel where thing is connected
+# NOTE: Scenario's values can be changed with environment variables predefined in defaults
+# Set environment variables to connect to Mainflux instance:
+# MF_MQTT_ENPOINT = IP/domain of MQTT endpoint on Mainflux
+# MF_MQTT_PORT = Port of MQTT endpoint on Mainflux
+# MF_THING_1_ID = First thing ID of pre-provisioned Mainflux thing
+# MF_THING_1_KEY = First thing key of pre-provisioned Mainflux thing
+# MF_THING_2_ID = Second thing ID of pre-provisioned Mainflux thing
+# MF_THING_2_KEY = Second thing key of pre-provisioned Mainflux thing.
+# MF_CHANNEL_ID = ID of pre-provisioned channel on which both things are connected
 #######
 
+defaults(
+        "PUB_NUM" = 10000,
+        "PUB_TIME" = 5,
+        "PUB_RATE" = 1,
+        "MSG_SIZE" = 100,
+        "QOS" = 2,
+        "MF_MQTT_ENDPOINT" = "127.0.0.1",
+        "MF_MQTT_PORT" = 1883
+)
+
```

Comment:
One question - do these scenarios have local vars? If yes, I think it would be good to define the ones we use at the top of the file, like:
```erlang
port = numvar("MF_MQTT_PORT"),
username = var("MF_THING_1_ID"),
password =  var("MF_THING_1_KEY")
```
and so on, so that you can see immediately all the things that you need to define in ENV at one place, rather than going through the file.

Reply:
No, there is no local variables.
But, if scenario is started from UI, environmental variables will be collected from the scenario and displayed in form so user can fill them.

Reply:
What if I do not use UI :). OK, strange that it does not have kind of local vars, but never mind, it was ust for the code readability.

Reply:
If you don't use UI and you don't define env vars, benchmark will crash as you already quoted. Like this:
```09:48:40.653 [error] Worker <0.240.0> on '1588-578519591169@ivke-ThinkPad-T580' has crashed: {var_is_unbound,"MF_THING_1_ID"}```